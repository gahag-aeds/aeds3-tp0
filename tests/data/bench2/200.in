200 100
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 13 41 92 89 1 60 41 43 90 41 86 55 18 20 39 30 63 0 42 32 12 43 23 39 21 75 41 29 84 41 27 50 34 71 91 35 31 32 78 22 73 16 77 43 36 68 25 0 68 67 
Add 123 188
Max 132 197
Sub 32 97
Max 38 103
Add 29 94
Min 53 118
Add 45 110
Sum 117 182
Sub 92 157
Min 107 172
Sub 96 161
Max 30 95
Add 79 144
Max 117 182
Add 115 180
Max 112 177
Sub 58 123
Min 85 150
Sub 86 151
Min 85 150
Add 87 152
Min 57 122
Add 79 144
Max 58 123
Sub 76 141
Min 130 195
Sub 64 129
Sum 74 139
Sub 72 137
Max 68 133
Add 9 74
Min 10 75
Add 15 80
Sum 40 105
Add 131 196
Max 128 193
Sub 92 157
Max 41 106
Sub 110 175
Max 134 199
Add 97 162
Max 21 86
Add 113 178
Sum 42 107
Add 3 68
Max 43 108
Add 103 168
Sum 11 76
Add 127 192
Sum 5 70
Add 95 160
Min 29 94
Add 11 76
Sum 123 188
Sub 36 101
Sum 87 152
Sub 68 133
Min 99 164
Sub 110 175
Max 89 154
Add 33 98
Max 118 183
Sub 48 113
Sum 47 112
Sub 108 173
Max 44 109
Sub 40 105
Sum 15 80
Sub 34 99
Max 99 164
Sub 98 163
Min 131 196
Sub 120 185
Max 76 141
Sub 122 187
Min 72 137
Add 119 184
Max 90 155
Sub 82 147
Sum 61 126
Sub 44 109
Sum 126 191
Sub 90 155
Min 5 70
Add 115 180
Sum 125 190
Add 91 156
Min 132 197
Sub 90 155
Sum 16 81
Add 87 152
Min 72 137
Add 83 148
Max 134 199
Sub 118 183
Min 56 121
Sub 44 109
Max 108 173
Add 21 86
Min 77 142
