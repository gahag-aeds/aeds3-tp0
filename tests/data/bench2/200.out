93
97
2
2945
0
99
91
92
0
-1
1
100
0
3537
97
0
3251
93
100
93
100
3263
101
3070
3116
1
3161
3358
0
95
95
3330
101
3182
94
1
93
-3
92
3131
3062
3
3023
1
3124
-5
93
-4
92
-5
